tailwind.config.js

```diff
@@ -19,6 +19,7 @@ export default {
 			'light-blue': '#ede7fb',
 			blue: '#3b00b9',
 			'blue-ribbon': '#0066ff',
+			'blue-ribbon-rgb': '0, 102, 255',
```

Comment:
Just sharing an idea, would following work?

1. Remove `blue-ribbon`
2. Then in CSS:

```
--color-blue-ribbon: rgb(theme(colors.blue-ribbon-rgb));
```

Reply:
not only it works, but even better:

```scss
	--color-primary-rgb: var(--color-blue-ribbon-rgb);
	--color-primary: rgb(var(--color-primary-rgb));
```